assets/css/hazard-card.css

```diff
@@ -0,0 +1,63 @@
+.sb-cards-btn--back {
+  display: flex;
+  align-items: center;
+  color: var(--neutral-neutral-900);
```

Comment:
Could you double-check the variables in our file _variables.css, in figma usually they repeat the same word twice, not sure why, but in our variables this would be just --neutral-900